### PR TITLE
css-layout-simple

### PR DESCRIPTION
**Pull request by doboapps**

### CODE REVIEW

**Review by Valcis — COMMENTED** [review]
great job! fresh code

**Comment by doboapps** [comment]
thank you! 😃

File: staff/doboapps/css-layout-simple/adaptive/styles/mobile.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+html,body{
```
Comment (manuelbarzi) [inline staff/doboapps/css-layout-simple/adaptive/styles/mobile.css]: TIP some css rules repeat in mobile and desktop. may you group them in main.css.

Comment (doboapps) [reply staff/doboapps/css-layout-simple/adaptive/styles/mobile.css]: Thank you for the observation. It is already changed. 😅

File: staff/doboapps/css-layout-simple/adaptive/index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Adaptative</title>
+    <!-- <link rel="stylesheet" href="styles/mobile.css"> -->
```
Comment (agandia9) [inline staff/doboapps/css-layout-simple/adaptive/index.html]: caution with comments... delete all before push!

File: staff/doboapps/js/challenges/to-camel-case/scripts/toCamelCase.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    if (typeof str !== 'string')
+    throw Error('input str is not a string');
+
+
+    var sentence = "";
+    str = str.toLowerCase();
+    
+    var myArray = str.split(" ");
+
+    myArray.map(word => {
```
Comment (agandia9) [inline staff/doboapps/js/challenges/to-camel-case/scripts/toCamelCase.js]: =>????.... this is ES5!

File: staff/doboapps/portfolio/index.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Portfolio</title>
+    <!-- <link rel="stylesheet" href="styles/mobile.css"> -->
```
Comment (agandia9) [inline staff/doboapps/portfolio/index.html:9]: delete commeeeeeents...

Comment (doboapps) [reply staff/doboapps/portfolio/index.html:9]: done! Thank you!

**Pull request merged**